catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -191,14 +190,14 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
       SuggesterResponse suggesterResponse = solrResponse.getSuggesterResponse();
 
       if (suggesterResponse != null) {
-        List<String> suggestionResults =
+        List<List<String>> suggestionResults =
```

Comment:
What is List<String> ?  Seems like something with maybe a hidden understanding that x.get(0) is an id and x.get(1) is a label.  Seems like a MapEntry that did id=name would be more obvious, or even a Map where Suggester.ID and Suggester.NAME get set in the map

Reply:
👍